tools/topology/topology2/cavs/platform/intel/hdmi-generic.conf

```diff
@@ -0,0 +1,152 @@
+Object.Dai {
```

Comment:
@bardliao can we reuse the cavs-passthrough-hdmi.conf inbstead of adding a new one?

Reply:
Can you please make it part of this PR?

Reply:
> Can you please make it part of this PR?

done